src/search-manager/messages.ts

```diff
@@ -105,6 +105,31 @@ const messages = defineMessages({
     defaultMessage: 'Video',
     description: 'Name of the "Video" component type in Studio',
   },
+  'blockType.choiceresponse': {
+    id: 'course-authoring.course-search.blockType.choiceresponse',
+    defaultMessage: 'Multiple Choice',
+    description: 'Name of the "choiceresponse" component type in Studio',
+  },
+  'blockType.multiplechoiceresponse': {
+    id: 'course-authoring.course-search.blockType.multiplechoiceresponse',
+    defaultMessage: 'Checkboxes',
+    description: 'Name of the "multiplechoiceresponse" component type in Studio',
+  },
```

Comment:
I think you are missing a description for `formularesponse` - see the screenshot in your PR description.